src/track.h

```diff
@@ -47,7 +47,7 @@ void tracking_channels_missed_update_error(u32 channels_mask);
 /* State management interface */
 bool tracker_channel_available(tracker_channel_id_t id, gnss_signal_t sid);
 bool tracker_channel_init(tracker_channel_id_t id, gnss_signal_t sid,
-                          u32 start_sample_count, float code_phase,
```

Comment:
I noticed this, it didnt match the variable name in track.c's `tracker_channel_init`